lib/class-wp-theme-json-gutenberg.php

```diff
@@ -141,77 +141,72 @@ class WP_Theme_JSON_Gutenberg {
 	 *
 	 * This contains the necessary metadata to process them:
 	 *
-	 * - path          => where to find the preset within the settings section
-	 *
-	 * - value_key     => the key that represents the value
-	 *
-	 * - value_func    => optionally, instead of value_key, a function to generate
-	 *                    the value that takes a preset as an argument
-	 *
-	 * - css_var_infix => infix to use in generating the CSS Custom Property. Example:
-	 *                   --wp--preset--<preset_infix>--<slug>: <preset_value>
-	 *
-	 * - classes      => array containing a structure with the classes to
-	 *                   generate for the presets. Each class should have
-	 *                   the class suffix and the property name. Example:
-	 *
-	 *                   .has-<slug>-<class_suffix> {
-	 *                       <property_name>: <preset_value>
-	 *                   }
+	 * - path       => where to find the preset within the settings section
+	 *
+	 * - value_key  => the key that represents the value
+	 *
+	 * - value_func => optionally, instead of value_key, a function to generate
+	 *                 the value that takes a preset as an argument
+	 *
+	 * - css_var    => name of the var to generate. The "$slug" substring will be
```

Comment:
Are we losing a bit of consistency by introducing this config. Meaning nothing would prevent presets from having `--rnadom-variable` instead of always starting with `--wp-preset`?

Reply:
Not really. The name of the variable is still controlled by the framework. It's just clearer what they are.